src/linear_algebra/basic.lean

```diff
@@ -291,13 +291,6 @@ instance : module R (M →ₗ[R] M₂) :=
 module.of_core $ by refine { smul := (•), ..};
   intros; ext; simp [smul_add, add_smul, smul_smul]
 
-/-- Composition by `f : M₂ → M₃` is a linear map from the space of linear maps `M → M₂` to the space of
```

Comment:
@urkud I would prefer if you leave `congr_right` in place in this PR. Please remove it in the same PR as you replace it.
Otherwise, I think this PR is ready to go.

Reply:
I definitely don't like the name `congr_right`, because there is no `equiv` involved. `comp_right`?

Reply:
Yup, sounds good. We should make sure that this change ends up in the git logs. (So the renaming should appear in the commit title of a git commit.)

Reply:
Done (not tested).